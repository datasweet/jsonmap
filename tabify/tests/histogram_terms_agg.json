{
  "took": 98,
  "timed_out": false,
  "_shards": {
    "total": 5,
    "successful": 5,
    "failed": 0
  },
  "hits": {
    "total": 768068,
    "max_score": 0,
    "hits": []
  },
  "aggregations": {
    "2": {
      "buckets": [
        {
          "4": {
            "doc_count_error_upper_bound": -1,
            "sum_other_doc_count": 186054,
            "buckets": [
              {
                "3": {
                  "value": 129913
                },
                "key": "_PRENOMS_RARES",
                "doc_count": 767
              },
              {
                "3": {
                  "value": 36853
                },
                "key": "LÉA",
                "doc_count": 383
              },
              {
                "3": {
                  "value": 33469
                },
                "key": "LUCAS",
                "doc_count": 385
              },
              {
                "3": {
                  "value": 31912
                },
                "key": "THÉO",
                "doc_count": 384
              },
              {
                "3": {
                  "value": 31572
                },
                "key": "THOMAS",
                "doc_count": 383
              }
            ]
          },
          "key": 2000,
          "doc_count": 188356
        },
        {
          "4": {
            "doc_count_error_upper_bound": -1,
            "sum_other_doc_count": 254761,
            "buckets": [
              {
                "3": {
                  "value": 205893
                },
                "key": "_PRENOMS_RARES",
                "doc_count": 959
              },
              {
                "3": {
                  "value": 38138
                },
                "key": "ENZO",
                "doc_count": 480
              },
              {
                "3": {
                  "value": 35994
                },
                "key": "LUCAS",
                "doc_count": 480
              },
              {
                "3": {
                  "value": 34296
                },
                "key": "EMMA",
                "doc_count": 480
              },
              {
                "3": {
                  "value": 31166
                },
                "key": "NATHAN",
                "doc_count": 478
              }
            ]
          },
          "key": 2005,
          "doc_count": 257638
        },
        {
          "4": {
            "doc_count_error_upper_bound": -1,
            "sum_other_doc_count": 266780,
            "buckets": [
              {
                "3": {
                  "value": 264390
                },
                "key": "_PRENOMS_RARES",
                "doc_count": 960
              },
              {
                "3": {
                  "value": 29881
                },
                "key": "LUCAS",
                "doc_count": 479
              },
              {
                "3": {
                  "value": 27309
                },
                "key": "NATHAN",
                "doc_count": 480
              },
              {
                "3": {
                  "value": 26791
                },
                "key": "EMMA",
                "doc_count": 476
              },
              {
                "3": {
                  "value": 24937
                },
                "key": "GABRIEL",
                "doc_count": 477
              }
            ]
          },
          "key": 2010,
          "doc_count": 269652
        },
        {
          "4": {
            "doc_count_error_upper_bound": -1,
            "sum_other_doc_count": 51877,
            "buckets": [
              {
                "3": {
                  "value": 56107
                },
                "key": "_PRENOMS_RARES",
                "doc_count": 192
              },
              {
                "3": {
                  "value": 5147
                },
                "key": "JULES",
                "doc_count": 96
              },
              {
                "3": {
                  "value": 5061
                },
                "key": "GABRIEL",
                "doc_count": 77
              },
              {
                "3": {
                  "value": 4540
                },
                "key": "LOUISE",
                "doc_count": 96
              },
              {
                "3": {
                  "value": 4377
                },
                "key": "ADAM",
                "doc_count": 84
              }
            ]
          },
          "key": 2015,
          "doc_count": 52422
        }
      ]
    }
  },
  "status": 200
}