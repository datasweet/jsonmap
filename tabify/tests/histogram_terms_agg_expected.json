[
  { "2": 2000, "2#doc_count": 188356, "2#4": "_PRENOMS_RARES", "2#4#doc_count": 767, "2#4#3": 129913 },
  { "2": 2000, "2#doc_count": 188356, "2#4": "LÉA", "2#4#doc_count": 383, "2#4#3": 36853 },
  { "2": 2000, "2#doc_count": 188356, "2#4": "LUCAS", "2#4#doc_count": 385, "2#4#3": 33469 },
  { "2": 2000, "2#doc_count": 188356, "2#4": "THÉO", "2#4#doc_count": 384, "2#4#3": 31912 },
  { "2": 2000, "2#doc_count": 188356, "2#4": "THOMAS", "2#4#doc_count": 383, "2#4#3": 31572 },
  { "2": 2005, "2#doc_count": 257638, "2#4": "_PRENOMS_RARES", "2#4#doc_count": 959, "2#4#3": 205893 },
  { "2": 2005, "2#doc_count": 257638, "2#4": "ENZO", "2#4#doc_count": 480, "2#4#3": 38138 },
  { "2": 2005, "2#doc_count": 257638, "2#4": "LUCAS", "2#4#doc_count": 480, "2#4#3": 35994 },
  { "2": 2005, "2#doc_count": 257638, "2#4": "EMMA", "2#4#doc_count": 480, "2#4#3": 34296 },
  { "2": 2005, "2#doc_count": 257638, "2#4": "NATHAN", "2#4#doc_count": 478, "2#4#3": 31166 },
  { "2": 2010, "2#doc_count": 269652, "2#4": "_PRENOMS_RARES", "2#4#doc_count": 960, "2#4#3": 264390 },
  { "2": 2010, "2#doc_count": 269652, "2#4": "LUCAS", "2#4#doc_count": 479, "2#4#3": 29881 },
  { "2": 2010, "2#doc_count": 269652, "2#4": "NATHAN", "2#4#doc_count": 480, "2#4#3": 27309 },
  { "2": 2010, "2#doc_count": 269652, "2#4": "EMMA", "2#4#doc_count": 476, "2#4#3": 26791 },
  { "2": 2010, "2#doc_count": 269652, "2#4": "GABRIEL", "2#4#doc_count": 477, "2#4#3": 24937 },
  { "2": 2015, "2#doc_count": 52422, "2#4": "_PRENOMS_RARES", "2#4#doc_count": 192, "2#4#3": 56107 },
  { "2": 2015, "2#doc_count": 52422, "2#4": "JULES", "2#4#doc_count": 96, "2#4#3": 5147 },
  { "2": 2015, "2#doc_count": 52422, "2#4": "GABRIEL", "2#4#doc_count": 77, "2#4#3": 5061 },
  { "2": 2015, "2#doc_count": 52422, "2#4": "LOUISE", "2#4#doc_count": 96, "2#4#3": 4540 },
  { "2": 2015, "2#doc_count": 52422, "2#4": "ADAM", "2#4#doc_count": 84, "2#4#3": 4377 }
]
